### PR TITLE
Add websocket handling

### PR DESCRIPTION
**Pull request by OskarAhl**

### CODE REVIEW

File: src/common/websocket/config.js [hunk 1]
```diff
@@ -6,64 +6,62 @@
  * git update-index --assume-unchanged src/javascript/config.js
  *
  */
-const domain_app_ids = {
-    // these domains as supported "production domains"
-    'deriv.com': 11780,
+const domain_config = {
```
Comment (ashkanx) [inline src/common/websocket/config.js]: I think production & staging don't matter here, as users won't be able to log in. also, there will be the same redirect url.

File: src/common/websocket/config.js [hunk 2]
```diff
@@ -6,64 +6,62 @@
...
 
 const getSocketURL = () => {
     let server_url = window.localStorage.getItem('config.server_url')
     if (!server_url) {
-        const loginid = window.localStorage.getItem('active_loginid')
-        const is_real = loginid && !/^VRT/.test(loginid)
-        const server = isProduction() && is_real ? 'green' : 'blue'
-
-        server_url = `${server}.binaryws.com`
+        server_url = `blue.binaryws.com`
```
Comment (ashkanx) [inline src/common/websocket/config.js]: let's connect to default or green

File: src/components/layout/header.js [hunk 3]
```diff
@@ -35,7 +36,7 @@ const Header = ({ siteTitle }) => (
                     padding: `0.5rem`,
                 }}
             >
-                Home
+                <Localize i18n_text="Home" />
```
Comment (ashkanx) [inline src/components/layout/header.js]: what an ugly syntax 😃

File: src/common/websocket/config.js (hunk 1 above)
Comment (OskarAhl) [reply src/common/websocket/config.js]: @ashkanx The app id is used for the websocket connection and we need the hostname to match `app_id` with environment

File: src/common/websocket/config.js (hunk 2 above)
Comment (OskarAhl) [reply src/common/websocket/config.js]: ok 👍

File: src/components/layout/header.js (hunk 3 above)
Comment (OskarAhl) [reply src/components/layout/header.js]: Got a more beautiful suggestion? 😅 Either way can change it later when we add translations

Comment (zakibinary) [reply src/components/layout/header.js]: Maybe just `text` instead of `i18n_text`?

Comment (OskarAhl) [reply src/components/layout/header.js]: @zaki yeah `text` is cleaner but I think we want to keep it more specific incase we need a extract translations script?

Comment (ashkanx) [reply src/components/layout/header.js]: `translate`?

**Pull request merged**